pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -179,7 +179,10 @@
     private String tlsTrustCertsFilePath = "";
     // Accept untrusted TLS certificate from client
     private boolean tlsAllowInsecureConnection = false;
-
+    // Specify the tls protocols and cipher the broker will use to negotiate during TLS Handshake.
```

Comment:
add to `broker.conf`, with documentation and examples for both parameters

Reply:
and in `proxy.conf` as well? should we have any test for this PR?

Reply:
Added